15 - LocalStorage/index-START.html

```diff
@@ -26,13 +26,58 @@ <h2>LOCAL TAPAS</h2>
   </div>
 
 <script>
-  const addItems = document.querySelector('.add-items');
-  const itemsList = document.querySelector('.plates');
-  const items = [];
+  class customStoredList {
```

Comment:
classes are typically camel cased (starting with upper case).

Reply:
Oops!! my bad! totally forgot about it!